wrappers.go

```diff
@@ -68,6 +69,42 @@ func (i internalObjectWrapper) PrimaryKey() []byte {
 	return i.pk
 }
 
+func (i internalObjectWrapper) Marshal() (bz []byte, err error) {
```

Comment:
@fdymylja , what's the point of the internalObjectWrapper?  Any object we're given will implement the [types.Object](https://github.com/iov-one/cosmos-sdk-crud/blob/stargate/internal/store/types/types.go#L18) interface, which has the key accessor methods, so why wrap it?  In fact, stripping the external object's `codec.ProtoMarshaler` interface from the internal object's interface just adds overhead and makes life more difficult doesn't it?